README.md

```diff
@@ -25,6 +25,23 @@ bus.on('message', function (data) {
 });
 ```
 
+## Quick Start
+
+1. Install
+
+```bash
+npm install --save framebus
+```
+
+2. Install typescript definitons for typescript projects.
```

Comment:
This makes it look like installing the typescript definition is _required_ to use the project, which is not the case for general use.